src/config/ConfigManager.cpp

```diff
@@ -738,7 +739,7 @@ void CConfigManager::postConfigReload(const Hyprlang::CParseResult& result) {
         g_pHyprOpenGL->m_bReloadScreenShader = true;
 
     // parseError will be displayed next frame
-    if (result.error && !std::any_cast<Hyprlang::INT>(m_pConfig->getConfigValue("debug:suppress_errors")))
+    if (result.error)
```

Comment:
why remove this?

Reply:
I meant to move it to HyprError to just suppress the drawing of the notification, but it appears I didn't do that? lol.
Does it make more sense to leave the suppress_error here as-is and move the 'extra' copy of the error string to ConfigManager and have hyprctl fetch it from there instead?

Reply:
leave it here absolutely, let plugins still use it even if disabled for config

Reply:
Ok, restored it there and moved the "permanent" copy of the current error messages to the ConfigManager